docs/customize/notifications.md

```diff
@@ -0,0 +1,146 @@
+                "self_html": "<link to the community>",
+            }
+        },
+        "topic": {
+            "id": "49e72dea-f14a-4774-a52d-a3d581fed86e",
+            "metadata": {
+                "title": "My Submitted Record",
+            },
+        }
+    }
+}
+```
+
+```jinja
+{# notifications/community-submission.submitted.jinja #}
+
+{%- block subject -%}
+New record submission for your community {{ notification.context.get("request").get("receiver").get("metadata").get("title") }} submitted by {{ notification.context.get("request").get("created_by").get("username") }}
+{%- endblock subject -%}
+
+{%- block html_body -%}
+<p>The record "{{ notification.context.get("request").get("topic").get("metadata").get("title") }}" was submitted to your community {{ notification.context.get("request").get("receiver").get("metadata").get("title") }} by {{ notification.context.get("request").get("created_by").get("username") }}.</p>
+
+<a href="{{ notification.context.get("request").get("links").get("self_html") }}" class="button">Review the request</a>
+{%- endblock html_body -%}
+
+{%- block plain_body -%}
+The record "{{ notification.context.get("request").get("topic").get("metadata").get("title") }}" was submitted to your community {{ notification.context.get("request").get("receiver").get("metadata").get("title") }} by {{ notification.context.get("request").get("created_by").get("username") }}.
+
+Review the request: {{ notification.context.get("request").get("links").get("self_html") }}
+{%- endblock plain_body -%}
+```
+
+We are pretty happy with most of the content, but want to override the `html_body` to include a link to the community and all requests.
+To do this, we will create a file with the same name and extend the base template.
+
+```jinja
+
+{% extends "invenio_notifications/community-submission.submitted.html" %}
+
+{% block html_body %}
+
+<p>The record "{{ notification.context.get("request").get("topic").get("metadata").get("title") }}" was submitted to your community {{ notification.context.get("request").get("receiver").get("metadata").get("title") }} by {{ notification.context.get("request").get("created_by").get("username") }}.</p>
+
+<a href="{{ notification.context.get("request").get("receiver").get("links").get("self_html") }}" class="button">Check out the community"</a>
+
+<a href="{{ notification.context.get("request").get("links").get("self_html") }}" class="button">Review the request</a>
+
+<a href="{{ notification.context.get("request").get("receiver").get("links").get("self_html") + "/requests" }}" class="button">Check out all community requests</a>
+
+
+{% endblock %}
+```
+
+## NOTIFICATIONS_BACKENDS
+
+This config variable allows to specify the available backends.
+For instance, you can provide an implementation of your institute specific tool of communication and send notifications via this backend. For this, simply extend the `NotificationBackend` class and implement the `send` method.
```

Comment:
Mainly to use the same terminology than in the example below (i.e. institute vs institution)
```suggestion
For instance, you can provide an implementation for your institution's preferred communication tool and send notifications via this backend. For this, simply extend the `NotificationBackend` class and implement the `send` method.
```